VL.Data/Interfaces/IBookService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using VL.Shared.Model;
+
+namespace VL.Shared.Interfaces
+{
+    public interface IBookService
+    {
+        public Task<List<Book>> GetBooksAsync();
+        public Task<Book?> GetBookAsync(int id);
+
+        public Task<Book?> UpdateBookAsync(Book book);
+
+        public Task<Book> CreateBookAsync(Book book);
+
+        public Task<bool> DeleteBookAsync(int id);
```

Comment:
Nothing wrong with what you did here but if its a BookService and the method is GetBookAsync() and the method signature returns a Book - I would argue its more verbose than it needs to be

var book = _bookService.GetAsync(1); 

 I know will return a book because I can see its from the book service, I can hover over GetAsync and see the return type, and Visual Studio / Resharper is going to decorate the var book with the type "Book"

Reply:
Did this make sense?

**_bookService**.GetAsync(1)

vs.

**_bookService**.GetBookAsync(1)

Reply:
yes

Reply:
Ok, comment with something or use an emoji when someone takes the time to type out something like that on future PRs - resolving with no response can be misinterpreted